UNRELEASED.md

```diff
@@ -1 +1,9 @@
 # Unreleased
+
+**Added:**
+
+- BpkCalendar
+  - Added abilitty to update the selected dates programmatically using `updateSelection` in BpkCalendarControler.
+  - BpkCalendar is can now be extended.
```

Comment:
spelling:
- `abilitty -> `ability`
- `BpkCalendarControler -> `BpkCalendarController`
- `is can` -> `can`

Reply:
👍 I'll fix this when I release and move this entry to the changelog.